ray-operator/config/samples/ray-cluster.autoscaler-v2.yaml

```
@@ -14,6 +14,10 @@ spec:
     # Optionally specify the Autoscaler container's securityContext.
     securityContext: {}
     env: []
+    # AUTOSCALER_UPDATE_INTERVAL_S is used to control how often the Autoscaler sidecar checks the cluster state
```

In K8s 1.29, "sidecar container" has a clear definition. It's a special case of init containers. Autoscaler is an application container instead of an init container.

Ref: https://arc.net/l/quote/cjbnurud

> Kubernetes implements sidecar containers as a special case of [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/);

```suggestion
    # AUTOSCALER_UPDATE_INTERVAL_S is used to control how often the Autoscaler container checks the cluster state
```